docs/sources/permissions/organization_roles.md

```diff
@@ -10,32 +10,42 @@ parent = "permissions"
 weight = 30
 +++
 
-# Organization Roles
+# Organization roles
 
 Users can belong to one or more organizations. A user's organization membership is tied to a role that defines what the user is allowed to do
 in that organization.
 
-## Admin Role
+## Admin role
 
 Can do everything scoped to the organization. For example:
 
-- Add and Edit data sources.
-- Add and Edit organization users and teams.
-- Configure App plugins and set org settings.
+- Can add, edit, and delete data sources.
+- Can add and edit organizations, users, and teams.
```

Comment:
Your change makes it so that with Grafana admin you can edit organization but that's not true you'll need [grafana admin flag](https://grafana.com/docs/grafana/latest/permissions/overview/#grafana-admin) for that. 

```suggestion
- Can add and edit users and teams in organizations.
```